src/service/eligibleService.js

```
@@ -560,6 +560,18 @@ class EligibleService {
     }
   }
 
+  // By midwife
+  static async deleteEligibleUserByMidwife(userId) {
+    try {
+      const response = await axiosInstance.delete(
+        `/eligible/midwife/delete/${userId}`
+      );
+      return response.data;
+    } catch (error) {
+      throw error;
```

Biome

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)